store/query_jsonmode_test.go

```diff
@@ -285,6 +285,22 @@ var (
 			query:   JSONWhere("Meta.Rating").Gt(&ratingMid).JSONOrderByDesc("Meta.TotalReads"),
 			ordered: true,
 		},
+		// Indexing
```

Comment:
Just basic tests to make sure it works, do not test the benefits of indexing here.